docs/topics/Gateway.md

```diff
@@ -1,10 +1,10 @@
 # Gateways
 
-Gateways are Discord's form of real-time communication over secure WebSockets. Clients will receive events and data over the gateway they are connected to and send data over the REST API. The API for interacting with Gateways is complex and fairly unforgiving, therefore it's highly recommended you read _all_ of the following documentation before writing a custom implementation.
+Gateways are Discord's form of real-time communication over secure WebSockets. Clients will receive events and data over the Gateway they are connected to and send data over the REST API. The API for interacting with Gateways is complex and fairly unforgiving; therefore, it's highly recommended you read _all_ of the following documentation before writing a custom implementation.
```

Comment:
why is this capitalized?

Reply:
Consistency, mostly I capitalized Gateway because it defines a more specific term, but I will revert this - Thanks 👍

Reply:
```suggestion
Gateways are Discord's form of real-time communication over secure WebSockets. Clients will receive events and data over the gateway they are connected to and send data over the REST API. The API for interacting with Gateways is complex and fairly unforgiving; therefore, it's highly recommended you read _all_ of the following documentation before writing a custom implementation.
```